dask_glm/families.py

```diff
@@ -59,3 +60,40 @@ def gradient(Xbeta, X, y):
     @staticmethod
     def hessian(Xbeta, X):
         return 2 * dot(X.T, X)
+
+
+class Poisson(object):
+    """
+    This implements Poisson regression for count data. 
+    See https://en.wikipedia.org/wiki/Poisson_regression.
+    """
+
+    @staticmethod
+    def loglike(Xbeta, y):
+        eXbeta = exp(Xbeta)
+        yXbeta = y*Xbeta
+        return (yXbeta - eXbeta).sum()
```

Comment:
Ah, actually this is probably the issue; what you have written appears to be the *true* log-likelihood (which makes sense considering we called this `loglike`).  In fact, this should be the *negative* log-likelihood because all of our functions attempt to minimize.

This is a failure on my part of documentation / naming.